source/2019-04-05-the-ember-times-issue-93.md

```diff
@@ -47,6 +47,20 @@ Side note 2. [If you haven't joined the Ember Community on Discord, do so today]
 
 ---
 
+## [Reason #2. Stable and More Reliable Than Ever 💪](https://emberjs.com/ember-community-survey-2019/#MS_Q401)
+
+According to the results of the latest community survey, a large proportion of the community has kept their Ember apps up to date with recent releases. Most Ember apps beyond 3.x were currently locked in at [version 3.7, 3.4 and 3.8](https://emberjs.com/ember-community-survey-2019/#MS_Q401).
```

Comment:
idk how much of a knit pick this is, or if my brain isn't awake, but "were currently" has confused me, cause it's like a "past tense present test" thing. idk -- just tripped me up :man_shrugging: